lib/app.js

```diff
@@ -0,0 +1,51 @@
+const net = require('net');
+const ChatRoom = require('./ChatRoom');
+const parsedMessage = require('./parseMessage');
+
+
+// pass createServer the callback 'connection' which gets fired everytime someone connects to the server
+const server = net.createServer((connection) => {
+    // console logs tha a user has connected
+    console.log('user connected');
```

Comment:
```suggestion
    console.log('user connected', connection.username);
```
This line may be helpful after line 18.